mamba_gator/envmanager.py

```diff
@@ -101,6 +101,22 @@ def parse_version(version: str) -> Optional[Version]:
         number = ord(letter) - ord('a') + 1
         version = f"{version[:-1]}.{number}"
 
+    # Handle custom versions like "custom" -> "0.0.0"
+    if version == "custom":
+        version = "0.0.0"
+        return Version(version)
+
+    # Handle 
+      # OpenSSL style versions like "1.1.1j" -> "1.1.1.post10"
+      # Legacy short form versions like "9d" -> "9.post4"
+    # Assuming letter suffix implies patch level
+    matchOpenSSL = re.match(r'^(\d+\.\d+\.\d+)([a-z])$', version)
+    matchLegacy = re.match(r'^(\d+)([a-z])$', version)
```

Comment:
Another nitpick (sorry!), I think these variable names should be snake_case instead of camelCase.

Reply:
Ah yes thank you Dan! I think sometimes I get stuck in JS land or Python land and it carries over to the other. I just pushed the update for this as well!